src/network/taraxa_capability.cpp

```diff
@@ -400,12 +400,19 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
       break;
     }
     case GetBlocksPacket: {
-      LOG(log_dg_dag_sync_) << "Received GetBlocksPacket";
+      LOG(log_dg_dag_sync_) << "Received GetBlocksPacket with " << _r.itemCount() << " known blocks";
+      set<blk_hash_t> known_non_finalized_blocks;
```

Comment:
maybe use `std::unordered_map` so insert and search is O(1)

Reply:
Why map ? as for insert& search complexity fix, unordered_set  should do it

Reply:
@JakubFornadel  changed it to unordered set.